src/App.scss

```diff
@@ -1,2 +1,4 @@
 @import "~@edx/edx-bootstrap/sass/edx/theme";
 @import "~bootstrap/scss/bootstrap";
+
+@import "~@edx/frontend-component-footer/src/lib/scss/site-footer";
```

Comment:
nit: Why blank line

Reply:
Separates by app scss we are pulling in. If we end up using more apps, the lines will help